src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -559,6 +570,32 @@ def _update_issue(self, new_comment=None, notify=True):
     # hotlist IDs.
     self.labels.remove_by_prefix('ReleaseBlock-')
 
+    # Special case: OSS-Fuzz "Reported" custom field.
+    added_reported = _get_labels(self.labels.added, 'Reported-')
+    if added_reported:
+      try:
+        # Assume there is only one entry.
+        year, month, day = _parse_date_label(added_reported[0])
+        custom_field_entries.append({
+            'customFieldId': _OSS_FUZZ_REPORTED_CUSTOM_FIELD_ID,
+            'dateValue': {
```

Comment:
Just curious- Where did you find that this was the way to update a "Date" customfield?

Reply:
Looks correct to me according to this example- https://yaqs.corp.google.com/eng/q/4249097869912440832

Reply:
Thanks! I was basing this off the service protobuf definitions, but I wasn't sure how the JSON transcoding worked.